autonomous/autonomous.py

```diff
@@ -311,14 +357,14 @@ def deposit_switch(self, initial_call):
             self.chassis.set_inputs(0, 0, 0)
             self.cube_number += 1
             self.cube_inside = False
+            self.deposited = False
+            self.lifter_automation.engage(initial_state='move_switch')
+        if not self.lifter_automation.is_executing():
```

Comment:
I'm surprised pyflakes doesn't actually use type hints. I bet pylint would warn on this. mypy should too if configured correctly, I think.